Formula/httping.rb

```diff
@@ -1,11 +1,10 @@
 class Httping < Formula
   desc "Ping-like tool for HTTP requests"
-  homepage "https://www.vanheusden.com/httping/"
-  url "https://www.mirrorservice.org/sites/distfiles.macports.org/httping/httping-2.5.tgz"
-  mirror "https://fossies.org/linux/www/httping-2.5.tgz"
-  sha256 "3e895a0a6d7bd79de25a255a1376d4da88eb09c34efdd0476ab5a907e75bfaf8"
-  license "GPL-2.0"
-  revision 2
+  homepage "https://github.com/folkertvanheusden/HTTPing"
+  url "https://github.com/folkertvanheusden/HTTPing/archive/refs/heads/master.zip"
```

Comment:
Homebrew formulae need a tagged release upstream, this download is going to change (and possibly break) with every commit. Could you request a tagged release?

Reply:
OK, i'll open an issue and see where I get...

Reply:
https://github.com/folkertvanheusden/HTTPing/issues/6

Reply:
> @SMillerDev: Homebrew formulae need a tagged release upstream, this download is going to change (and possibly break) with every commit. Could you request a tagged release?

Done.